docs/reference/commandline/volume_create.md

```diff
@@ -54,17 +54,25 @@ different volume drivers may do different things (or nothing at all).
 
 The built-in `local` driver on Windows does not support any options.
 
-The built-in `local` driver on Linux accepts options similar to the linux `mount`
-command:
+The built-in `local` driver on Linux accepts options similar to the linux `mount` command. You can provide multiple options using a comma-separated list.
```

Comment:
This is a bit confusing; from that sentence I could interpret that as both examples below are valid (are they?):

```
--opt type=tmpfs --opt device=tmpfs --opt o=size=100m,uid=1000
```

```
--opt type=tmpfs,device=tmpfs,o=size=100m,uid=1000
```